src/packages/solid/components/Tile/Tile.tsx

```diff
@@ -223,8 +224,9 @@ const Tile: Component<TileProps> = (props: TileProps) => {
           {props.bottom}
         </View>
       </Show>
-
-      <Show when={props.progressBar?.progress ? props.progressBar.progress > 0 : 0}>
+      {/* TODO why was this a ternary? */}
+      <Show when={props.progressBar.progress > 0}>
```

Comment:
progressBar can be undefineid- need `props.progressBar?.progress`

Reply:
i think we're still good to pull it out of the ternary though, right?